isthmus/src/test/java/io/substrait/isthmus/IsthmusEntryPointTest.java

```diff
@@ -12,12 +12,12 @@
 import picocli.CommandLine;
 import picocli.CommandLine.ParameterException;
 
-class PlanEntryPointTest {
+class IsthmusEntryPointTest {
 
   /** Test that the default values are set correctly into the {@link FeatureBoard}. */
   @Test
   void defaultFeatureBoard() {
-    PlanEntryPoint planEntryPoint = new PlanEntryPoint();
+    IsthmusEntryPoint planEntryPoint = new IsthmusEntryPoint();
```

Comment:
Should we also rename `planEntryPoint` to `isthmusEntryPoint`?

Reply:
Changed